api/pkg/block/volumeList.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 The OpenSDS Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package block
+
+import (
+	//"time"
+
+	//"bytes"
+	//"encoding/xml"
+	"github.com/emicklei/go-restful"
+	"github.com/opensds/multi-cloud/api/pkg/common"
+	"github.com/opensds/multi-cloud/api/pkg/policy"
+	block "github.com/opensds/multi-cloud/block/proto"
+	log "github.com/sirupsen/logrus"
+	"net/http"
+	//"strconv"
+)
+
+func (s *APIService) ListVolumes(request *restful.Request, response *restful.Response) {
+	if !policy.Authorize(request, response, "volume:list") {
```

Comment:
Is this check duplicate with the one in RouteVolumeList?

Reply:
Yes, removed